### PR TITLE
Update changelog in prep for 5.2.0 release candidate

### PR DESCRIPTION
**Pull request by aaronayres35**
[targeting `maint/5.2`]

The updates for 5.1.1 were not yet pulled into master, so they are added here as well.
This PR updates the changelog for the upcoming 5.2 release candidate. Any additional changes made after the candidate but before the release will be added in future PR if needed.

### CODE REVIEW

File: CHANGES.txt [hunk 1]
```diff
@@ -1,6 +1,73 @@
 Enable CHANGELOG
 ================
 
+Enable 5.2.0
+============
+
+Thanks to:
+
+*
```
Comment (rahulporuri) [inline CHANGES.txt:9]: Did you want to fill these in before merging the PR?

File: CHANGES.txt [hunk 2]
```diff
@@ -1,6 +1,73 @@
...
+
+Enhancements
+------------
+
+* Provide better support for mixed writing systems (#763, #764 #767, #768)
+
+Changes
+-------
+
+*
```
Comment (rahulporuri) [inline CHANGES.txt:19]: same here

File: CHANGES.txt [hunk 3]
```diff
@@ -1,6 +1,73 @@
...
+-------------
+
+* Convert # to #: to improve docstrings in markers (#784)
+* Document EnableTestAssistant (#800)
+
+Maintenance
+-----------
+
+* Remove "use_draw_order" code path (#777)
+* add useful objects to "enable.api" (#779, #788)
```
Comment (rahulporuri) [inline CHANGES.txt:39]: nit : capitalization. here and in a few other places.

**Review by rahulporuri — APPROVED** [review]
Almost LGTM.

File: CHANGES.txt (hunk 1 above)
Comment (aaronayres35) [reply CHANGES.txt:9]: 🤦 yep, whoops

**Review by rahulporuri — APPROVED** [review]
LGTM

**Pull request merged**